client/src/pages/Home.js

```diff
@@ -5,30 +5,38 @@ import tourPath from "../assets/tour.svg";
 import searchPath from "../assets/search.svg";
 import favoritePath from "../assets/favourites.svg";
 import PageHeadline from "../components/Header/PageHeadline";
+import { Link } from "react-router-dom";
 
 const HomeDiv = styled.div`
   display: flex;
   flex-direction: column;
   align-content: center;
-  height: 90vh;
+  width: 100%;
+  max-width: 600px;
 `;
 
 const Home = () => {
   return (
     <HomeDiv>
       <PageHeadline>HOME</PageHeadline>
-      <Card primary>
-        <h2>TOUR</h2>
-        <img src={tourPath} alt="touricon" />
-      </Card>
-      <Card primary>
-        <h2>SUCHE</h2>
-        <img src={searchPath} alt="searchicon" />
-      </Card>
-      <Card primary>
-        <h2>FAVORITEN</h2>
-        <img src={favoritePath} alt="favoriteicon" />
-      </Card>
+      <Link to="/choice">
+        <Card primary>
+          <h2>TOUR</h2>
+          <img src={tourPath} alt="touricon" />
+        </Card>
+      </Link>
+      <Link to="search">
```

Comment:
I think you missed a / here
```suggestion
      <Link to="/search">
```